server/catgenome/src/main/java/com/epam/catgenome/util/feature/reader/TabixIteratorLineReader.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.catgenome.util.feature.reader;
+import htsjdk.samtools.util.RuntimeIOException;
+import htsjdk.tribble.readers.LineReader;
+
+import java.io.IOException;
+
+/**
+ * @author Jim Robinson
+ * @date 2/11/12
+ */
+public class TabixIteratorLineReader implements LineReader {
```

Comment:
I think we don't need this class, it's a full copy of HTSJDK's TabixIteratorLineReader

Reply:
It is used in https://github.com/epam/NGB/pull/149/files#diff-490b452e67fc21898b01bbd24d566fb7R181 and underneath there is `TabixReader.Iterator`, so we can't use library class if we change `TabixReader`.